bin/configurable-http-proxy

```diff
@@ -25,6 +25,7 @@ args
     .option('--api-ssl-key <keyfile>', 'SSL key to use, if any, for API requests')
     .option('--api-ssl-cert <certfile>', 'SSL certificate to use, if any, for API requests')
     .option('--default-target <host>', 'Default proxy target (proto://host[:port]')
+    .option('--redirect-to-https-from-port <redirect-port>', 'Redirects traffic from redirect port to the port')
```

Comment:
I think it's okay to call this `--redirect-port`, and then clarify in the help string that it's `Redirect http requests on this port to the server on https`.